tools/rosetta/lib/internal/service/online.go

```diff
@@ -25,26 +23,16 @@ func NewOnlineNetwork(network *types.NetworkIdentifier, client crgtypes.Client,
 	ctx, cancel := context.WithTimeout(context.Background(), genesisBlockFetchTimeout)
 	defer cancel()
 
-	var genesisHeight int64 = -1 // to use initial_height in genesis.json
-	block, err := client.BlockByHeight(ctx, &genesisHeight)
+	var genesisHeight int64 = 1 // to get genesis block height
+	genesisBlock, err := client.BlockByHeight(ctx, &genesisHeight)
 	if err != nil {
-		return OnlineNetwork{}, err
+		logger.Error(fmt.Sprintf("Could not get genesis block height. %v", err))
```

Comment:
Just curious why we don't return an error anymore. Wouldn't `genesisBlock.Block` be 0 then?